python/ray/train/tests/test_wandb_integration.py

```diff
@@ -0,0 +1,80 @@
+"""
```

Comment:
Can we move this into `air/tests/test_integration_wandb`? It's confusing to have to separate files to test the same integration.

Reply:
hmmm, I actually find putting all tests about wandb integration in one file kind of confusing.

my understanding of `air/tests/test_integration_wandb`, is everything about testing the logger callback as well as mixin. Those are the fundamental primitives of the integration, thus in air/.

for this test, it's testing the e2e story of train integration with it - like if certain fields pertained to trainer will finally show up in wandb ui. (you can imagine there will be another one that tests the e2e tune integration with it as well - maybe with more emphasis on testing multiple trials' behavior)

imo dumping all of them into a single file feels crowded.